doc/dev/developer_guide/tests-unit-tests.rst

```diff
@@ -72,34 +71,65 @@ the `cram task`_.
 .. _`cram`: https://bitheap.org/cram/
 .. _`cram task`: https://github.com/ceph/ceph/blob/master/qa/tasks/cram.py
 
-Tox based testing of python modules
+Tox-based testing of Python modules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some of the Python modules in Ceph use `tox <https://tox.readthedocs.io/en/latest/>`_ 
+to run their unit tests.
+
+Most of these Python modules can be found in the directory ``./src/pybind/``.
+
+Currently (December 2020) the following modules use **tox**:
+
+* Cephadm (``./src/cephadm/tox.ini``)
+* Manager core (``./src/pybind/mgr``)
+
+  * ``./src/pybind/mgr/tox.ini``
+    
+  * ``./src/pybind/mgr/dashboard/tox.ini``
+
+  * ``./src/pybind/mgr/snap_schedule/tox.ini``
```

Comment:
snap_schedule is already included in the MGR. IMO it's easier to simply run  `tox snap_schedule` from the mgr dir.

Reply:
I've removed this from this list.